vars/main.yml

```diff
@@ -23,3 +23,11 @@ dnsdist_powerdns_repo_11:
   gpg_key_id: "9FAAA5577E8FCF62093D036C1B0C6205FD380FBB"
   yum_repo_baseurl: "http://repo.powerdns.com/centos/$basearch/$releasever/dnsdist-11"
   yum_repo_name: "powerdns-dnsdist"
+
+dnsdist_powerdns_repo_12:
+  apt_repo_origin: "repo.powerdns.com"
+  apt_repo: "deb [arch=amd64] http://repo.powerdns.com/{{ ansible_distribution | lower }} {{ ansible_distribution_release | lower }}-dnsdist-12 main"
```

Comment:
This'll break once we're going to support raspbian with this role (OK for now)

Reply:
Yes. Once we will extend the support to raspbian we should definitely fix it.